lib/elixir/lib/kernel/utils.ex

```diff
@@ -34,6 +35,7 @@ defmodule Kernel.Utils do
     as = Keyword.get(opts, :as, name)
     :lists.map(fn as_args ->
       formal_args = make_formal_args(as_args)
+      # TODO: Remove by 2.0 use support of :append_first option
```

Comment:
We can probably get rid of this TODO because we can deduct this by the first TODO. :)